src/main/scala/fft/FFTUtilities.scala

```diff
@@ -23,7 +21,7 @@ object BarrelShifter {
   def apply[T<:Data](in: Vec[T], shift: UInt): Vec[T] = 
   {
     Vec((0 until in.size).map(i => {
-      val idx = Wire(UInt(width=log2Up(in.size)))
+      val idx = Wire(UInt(width=log2Up(in.size).W))
```

Comment:
I think we should get rid of "width=" here.

Reply:
Yeah, it gives a deprecation warning atm.